lib/reportportal.rb

```diff
@@ -1,6 +1,8 @@
+require 'base64'
```

Comment:
do we need require `base64` and `mime/types`? It used to work before your changes.

Reply:
As you can see code used `Base64` and `MIME::Types` so it assumed they are required elsewhere.